src/schema/v2/artwork/__tests__/artwork.test.js

```diff
@@ -904,6 +904,88 @@ describe("Artwork type", () => {
     })
   })
 
+  describe("#canRequestLotConditionsReport", () => {
+    const query = `
+      {
+        artwork(id: "richard-prince-untitled-portrait") {
+          canRequestLotConditionsReport
+        }
+      }
+    `
+
+    it("is true if the artwork's sale is an auction, live, and has lot conditions report enabled", () => {
+      context.saleLoader = sinon.stub().returns(
+        Promise.resolve({
+          live: true,
+          is_auction: true,
+          lot_conditions_report_enabled: true,
+        })
+      )
+
+      return runQuery(query, context).then(data => {
+        expect(data).toEqual({
+          artwork: {
+            canRequestLotConditionsReport: true,
+          },
+        })
+      })
+    })
+
+    it("is false if the artwork's sale is not an auction", () => {
+      context.saleLoader = sinon.stub().returns(
+        Promise.resolve({
+          live: true,
+          is_auction: false,
+          lot_conditions_report_enabled: true,
+        })
+      )
+
+      return runQuery(query, context).then(data => {
+        expect(data).toEqual({
+          artwork: {
+            canRequestLotConditionsReport: false,
+          },
+        })
+      })
+    })
+
+    it("is false if the artwork is biddable", () => {
```

Comment:
I don't think this description is correct? Is `biddable` what this test checks?

Reply:
I'm not sure if it sounds better but just updated it to `"is false if the artwork's sale is an auction but has not started yet"`. Any other suggestions?

Reply:
That's OK